src/std_seq.md

```diff
@@ -1,6 +1,6 @@
 ## @std/seq to express Sequential Algorithms in Alan
 
-Alan does not allow arbitrary, or classical, loops or recursion and [the preferred way to do iteration is with arrays](./advanced_examples.md#loopln). However, there are algorithms people need to write that are inherently sequential, and often non-deterministic on the number of steps needed to take, such as with numeric approximation algorithms like Newton-Raphson, or anything written as a recursive function call. Most of the problems developers need to solve do not require this linearity, but for when it really is required, we are reintroducing this power in a controlled manner that still allows the AVM to be able to plan around these algorithms and provide the guarantee of termination. The syntax `@std/seq` provides was [designed (and only partially implemented so far)](https://github.com/alantech/alan/blob/main/rfcs/007%20-%20Sequential%20Algorithms%20RFC.md) to *not* feel like classical control flow in order to nudge developers to find better, more deterministic and more parallelizable mechanisms, instead.
+Alan does not allow arbitrary, or classical, loops or recursion and [the preferred way to do iteration is with arrays](./advanced_examples.md#loopln). However, there are algorithms people need to write that are inherently sequential, and often non-deterministic on the number of steps needed to take, such as with numeric approximation algorithms like Newton-Raphson, or anything written as a recursive function call. Most of the problems developers need to solve do not require this linearity, but for when it really is required, we are reintroducing this power in a controlled manner that still allows the AVM to be able to plan around these algorithms and provide the guarantee of termination. The syntax `@std/seq` provides was [designed](https://github.com/alantech/alan/blob/main/rfcs/007%20-%20Sequential%20Algorithms%20RFC.md) to *not* feel like classical control flow in order to nudge developers to find better, more deterministic and more parallelizable mechanisms, instead.
 
 Here are the function signatures of the sequential functions that `@std/seq` will provide:
```

Comment:
Some other future tense statements in here, eg 'will provide' should be 'provides'